src/ansys/platform/instancemanagement/client.py

```diff
@@ -263,3 +265,36 @@ def create_instance(
             )
         definition = definitions[0]
         return definition.create_instance(timeout=requests_timeout)
+
+    def get_instance(self, name: str, timeout: float = None) -> Instance:
+        """Get a remote product instance by name.
+
+        Parameters
+        ----------
+        name: str
+            Name of the instance to get (for example ``instances/mapdl-1212``).
+
+        Returns
+        -------
+        Instance
+            A remote instance.
+
+        timeout : float, optional
+            Maximum time in seconds for the request. The default is ``None``.
+
```

Comment:
doc for the timeout parameter to move in the Parameters section?

Reply:
Yes! thank you